### DISCUSSION TITLE
Alias for today/now in query

### DISCUSSION THREAD

**Discussion by igonro** [carltongibson/django-filter#1481] [Q&A] [answered]
I know this question may sound weird, but is there any way of implementing this easily?

For example, today is 2022-02-18, so if I make a query to `/myapi/?creation_datetime__date=_today_`, the API could translate the `_today_` alias to `2022-02-18`, or the same thing with `_now_` and translate to `2022-02-18 10:48:00`.

The idea is to have a fixed URL to make this dynamic query.

Thanks.

**Comment by carltongibson** [comment, marked as answer]
You could define... 

- a form field that converted the special values in `value_from_datadict` — I'd probably do this. 
- a filter that (doing similar) looked for the special value, before passing off to the superclass if (say) a date was provided. 
- define a custom `method`
- ... 

Lots of ways. 😄

**Reply by igonro** [reply]
Thanks!
